webapp/channels/src/components/announcement_bar/notification_permission_bar/index.tsx

```diff
@@ -19,13 +20,19 @@ import {
 
 export default function NotificationPermissionBar() {
     const isLoggedIn = Boolean(useSelector(getCurrentUserId));
+    const subscription = useSelector(getCloudSubscription);
 
     useDesktopAppNotificationPermission();
 
     if (!isLoggedIn) {
         return null;
     }
 
+    // Don't show the notification bar if it's a cloud preview environment
```

Comment:
Why aren't we showing this? It seems like it would be important for a user to have working notifications if they're trying out MM for the first time

Reply:
Previews will only last 1 hour, and are mostly meant for people to poke around in a pre seeded environment. Theres a persistent notification bar for previews with a timer countdown and a CTA, and we don't want this to conflict with it in any way. Stacking looks like a broken window imo.